synfig-studio/src/gui/docks/dock_canvases.cpp

```diff
@@ -76,7 +76,7 @@ Dock_Canvases::Dock_Canvases():
 	);
 
 	add_button(
-		Gtk::StockID("gtk-delete"),
+		("_Delete"),
```

Comment:
It isn't translatable this way (Although it's commented out :P)

Reply:
Yeah. I removed it here because I'm looking in the code for `Gtk::StockID` to make sure I've found all the places.
Also I didn't delete the comment because I didn't check it, maybe these buttons can be used in the future.

Reply:
Should macro `_()` be used anyway?

Reply:
Interesting question. I'm not sure if it makes any difference to the commented out code.
Anyway, after uncommenting it, this code needs to be reviewed.
But I can add it if you want.